packages/in_app_purchase/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -252,7 +304,8 @@ private void queryPurchases(String skuType, MethodChannel.Result result) {
       return;
     }
 
-    // Like in our connect call, consider the billing client responding a "success" here regardless of status code.
+    // Like in our connect call, consider the billing client responding a "success" here regardless
```

Comment:
is there any change here?

Reply:
Added a line break since the comment was long (autoformatted by 'google-java-format')